koin-projects/examples/android-mvvm-coroutines/src/main/kotlin/fr/ekito/myweatherapp/data/datasource/webservice/local/AndroidJsonReader.kt

```diff
@@ -1,25 +1,24 @@
-package fr.ekito.myweatherapp.data.datasource.local
+package fr.ekito.myweatherapp.data.datasource.webservice.local
 
-import android.app.Application
-import fr.ekito.myweatherapp.data.datasource.webservice.local.BaseReader
+import android.content.Context
 import java.io.BufferedReader
 import java.io.InputStreamReader
 
 /**
  * Read Json File from assets/json
  */
-class AndroidJsonReader(val application: Application) : BaseReader() {
+class AndroidJsonReader(val application: Context) : BaseReader() {
```

Comment:
naming should be context and not application

Reply:
yep